packages/demo/examples/08-network/GraphWithCustimizedRenderer.jsx

```diff
@@ -0,0 +1,165 @@
+/* eslint class-methods-use-this: 0, react/no-unused-prop-types: 0 */
```

Comment:
can you get rid of these exceptions (if you remove `getStateFromProps` `class-methods-use-this` shouldn't throw errors any more)